composer.json

```diff
@@ -67,11 +67,12 @@
   "repositories": [
     {
       "type": "path",
-      "url": "app"
-    },
-    {
-      "type": "git",
-      "url": "https://github.com/mautic/BazingaOAuthServerBundle.git"
+      "url": "app",
+      "options": {
+        "versions": {
+            "mautic/core-lib": "3.2.x-dev"
```

Comment:
The folks at Symfony [also do it this way](https://github.com/symfony/symfony/blob/f12a4190e174a8efa1830083729e0b1761a83c82/composer.json#L190-LL198). Don't ask me why or how it works 🙈 

Here's the official Composer documentation for this setting: https://getcomposer.org/doc/05-repositories.md#path

Reply:
CC @nickveenhof